docs/plugins/network/README.md

```diff
@@ -5,6 +5,19 @@ $ ionic cordova plugin add cordova-plugin-network-information
 $ npm install @awesome-cordova-plugins/network
 ```
 
+# How to import?
+Import it in your desired module e.g for AppModule:
+```
+import { Network } from '@ionic-native/network/ngx';
```

Comment:
Hi... maybe a typo here?
Would be...
import { Network } from '@awesome-cordova-plugins/network/ngx';